web/packages/teleterm/src/ui/ConnectMyComputer/DocumentConnectMyComputerStatus/DocumentConnectMyComputerStatus.tsx

```diff
@@ -215,7 +215,7 @@ function renderLabels(labelsList: tsh.Label[]): JSX.Element[] {
   ));
 }
 
-export function prettifyCurrentAction(currentAction: CurrentAction): {
```

Comment:
Yeah, I had it initially because I had the new implementation and the old one side by side and I didn't want ESLint to complain about the new one not being used anywhere. I should probably disable eslint with a comment instead.